### DISCUSSION TITLE
Scope 3 Cat 2 - New versus Pre-Owned Assets

### DISCUSSION THREAD

**Discussion by samblustein** [USEPA/supply-chain-factors#71] [Q&A]
Hi all,

I have a question about new versus previously existing buildings, specifically, but may be applicable to new versus pre-owned capital assets, generally. In the case of an acquired building that already existed, would the purchasing company include the emissions from the construction of these pre-owned assets in their Category 2 emissions? The GHG Protocol implies the answer is "yes," however, guidance from the [UK Green Building Council on Scope 3 Guidance for Commercial Real Estate (page 30)](https://www.ukgbc.org/wp-content/uploads/2019/07/Scope-3-guide-for-commercial-real-estate.pdf) recommends that upstream emissions from existing buildings are not transferred from seller to buyer at the point of sale (unless upstream emissions are readily available).

Any thoughts would be greatly appreciated! Thanks!

**Comment by bl-young** [comment]
Hi @samblustein - thanks for your question. We can't offer guidance on GHG reporting for questions like this, but maybe other members of the community might chime in.
